the-devastating-chickens/BangazonSite
# User can view List of Products for ProductType Details

User can view List of Products for ProductType Details

# Description

Modified the scaffolding for the ProductTypesController Details method. The method now sets the Products property of the propertyType to include all products of the product type selected. The product type is passed into the view.

Fixes Issue # [3](https://github.com/the-devastating-chickens/BangazonSite/issues/3)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. `git fetch --all`
1. `git checkout cm-wednesday`
1. Review the code within the ProductTypesController and make sure the comments are well written.
1. Run the application
1. Navigate to product types and select details of a few of the product types. (Tools should have a few listed below)



# Checklist:
